grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -757,9 +757,9 @@ class ShipmentService {
 
             def quantityAvailableToReturn
             if (withBinLocation) {
-                quantityAvailableToReturn = productAvailabilityService.getQuantityNotPickedInBinLocation(shipmentItem.inventoryItem, shipmentItem.binLocation)
+                quantityAvailableToReturn = productAvailabilityService.getQuantityNotPickedInBinLocation(shipmentItem.inventoryItem, shipmentItem.binLocation) + shipmentItem.quantity
```

Comment:
I think you should use quantity picked, not quantity,
right now in the shipmentItem there is method to get picked quantity for the requisition based shipment,
you could add similar method for the order

Reply:
Done